package.json

```diff
@@ -379,7 +382,8 @@
     "jest-axe/axe-core": "4.4.3",
     "eslint": "7.25.0",
     "@mdx-js/loader/loader-utils": "~2.0.4",
-    "swc-loader": "^0.2.3"
+    "swc-loader": "^0.2.3",
+    "puppeteer": "19.6.0"
```

Comment:
Had to pin puppeteer to the latest version we use because when v9 packages are transpiled with `swc`, the `perf-test-react-components` package completely [freezes](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=286566&view=logs&j=6239fe86-931d-5083-f545-2ca826ae20fe&t=d661239c-67af-5b1a-2911-8552b6610048). This is presumably caused by `flamegrill` using an ancient version of puppeteer. After this change, the v8 and v9 perf tests run as expected except for v8's `TeachingBubble` perf test which runs into a Navigation Timeout Error and causes another freeze. Mitigation to move this work forward is to temporarilty disable v8's TeachingBubble perf test